spec/steps/execute.fmf

```diff
@@ -13,6 +13,41 @@ description: |
     restraint
         use the restraint harness to execute beakerlib tests
 
+    For each ``plan`` the execute step should produce a ``results.yaml`` file
+    with the list of results for each test in the following format::
+
+        result: PLAN_RESULT
```

Comment:
At first sight it seemed to me as a good idea but then I realized: Which step should be responsible for aggregating and summarizing the results? Isn't this a task for the `report` step? To look through all test plans, aggregate results of individual tests and show the overall result?

One real life question: If the same test case is executed in multiple plans shall we aggregate the results when reporting to the user? For example running across all supported architectures.